Formula/crystal-lang.rb

```diff
@@ -6,25 +6,8 @@ class CrystalLang < Formula
   head "https://github.com/crystal-lang/crystal.git"
 
   stable do
```

Comment:
Cool, thanks. It scopes whatever is inside it to the particular software spec (stable, devel, or head).